apps/rush-lib/src/logic/VersionManager.ts

```diff
@@ -255,9 +256,17 @@ export class VersionManager {
             );
           }
           dependencies[updatedDependentProjectName] = newDependencyVersion;
+          const preferredVersions: Map<string, string> = this._rushConfiguration.getCommonVersions().preferredVersions;
+          if (preferredVersions.has(updatedDependentProjectName)) {
+            preferredVersions[updatedDependentProjectName] = newDependencyVersion;
```

Comment:
@Lincoded Use `Map.set()` to add members to a map.  Using the `[` `]` operator will modify the `Map` class instead of its contents.